src/components/application_manager/rpc_plugins/app_service_rpc_plugin/test/app_service_plugin/app_service_app_extension_test.cc

```diff
@@ -199,8 +199,8 @@ TEST_F(AppServiceAppExtensionTest, ProcessResumption_SUCCESS) {
   resumption_data[application_manager::strings::application_subscriptions]
                  [kAppServiceInfoKey] = app_service_data;
 
-  resumption::Subscriber subscriber;
-  app_service_app_extension_->ProcessResumption(resumption_data, subscriber);
+  //  resumption::Subscriber subscriber;
```

Comment:
can be removed?

Reply:
@iCollin removed in https://github.com/smartdevicelink/sdl_core/pull/3475/commits/8db46ca18fc194f0ae761acfe9b83ee480f72f4a